app/views/works/edit_multiple.html.erb

```diff
@@ -11,24 +11,24 @@
   <%= ts("Your edits will be applied to") %> <strong><%= ts("all") %></strong> <%= ts("of the following works:") %>
 </p>
 
-<%= render "works/work_abbreviated_list", :works => @works %>
+<%= render "works/work_abbreviated_list", works: @works %>
```

Comment:
I think these changes might've snuck in from #4655. Can you remove them from this pull request, since they're unrelated to AO3-6613?

Reply:
Oops. Good pick up, thank you! I've reverted this one now and will make sure it has made its way into #4655 shortly.